shared/getDeviceOnDash/artik5.md

```diff
@@ -1,22 +1,22 @@
-## Setting Up Your Device
+In order to get **resinOS** up and running on your {{ $device.name }}, you need to first set it up to boot from your {{ $device.bootMedia }} rather than its internal [eMMC memory][emmc-link]. To do this we need to set the tiny `SW2` dip switch to position `1:on` and `2:on`, as shown below.
 
-In order to get the resinOS up and running on our {{ $device.name }} we need to first set it up to boot from our {{ $device.bootMedia }} rather than its internal [eMMC memory][emmc-link]. To do this we need to set the tiny `SW2` dip switch to position `1:on` and `2:on`, as shown below.
-
-__Note:__ This resinOS will completely write over the existing eMMC.
+__Note:__ **ResinOS** will completely write over the existing eMMC.
 
 <img src="/img/artik5/artik5-dev-kit.png" width="80%">
 
-We can now insert the 5VDC power cable and flip the power, switch labelled `PWR SW`, to the `on` position. We should now have some glowing LEDs indicating a sign of life. Next we need to press and hold the `SW3 POWER` push button for 1 or 2 seconds, this starts the boot from the {{ $device.bootMedia }}.
+You can now insert the 5VDC power cable and flip the power, switch labelled `PWR SW`, to the `on` position. You should now have some glowing LEDs indicating a sign of life. Next, you need to press and hold the `SW3 POWER` push button for 1 or 2 seconds, this starts the boot from the {{ $device.bootMedia }}.
 
-After a few seconds you should see your {{ $device.name }} appear on the resin.io dashboard. If it still hasn't appeared after 1 or 2 minutes, double check your ethernet connection or that you entered the right wifi credentials at download time. If you still can't get it online, contact us on support.
+After a few seconds you should see your {{ $device.name }} appear on the **resin.io** dashboard. If it still hasn't appeared after 1 or 2 minutes, double check your ethernet connection or that you entered the right wifi credentials at download time. If you still can't get it online, contact us on support.
```

Comment:
The "If you still can't get it online, contact us on support." was removed in `artik10.md`, so should probably be removed here too?